Add plugin: Local LLM Helper

manimohans opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin: https://github.com/manimohans/obsidian-local-llm-helper

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [x]  macOS
  - [x]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
["minAppVersion": "0.15.0",](https://github.com/manimohans/obsidian-local-llm-helper/blob/a80440c68f9a3a0c4a455c8ccc2e6f1180868ac6/manifest.json#L5)
You are using newer API's, set this to the latest public build number.

["fundingUrl": "manee.eth",](https://github.com/manimohans/obsidian-local-llm-helper/blob/a80440c68f9a3a0c4a455c8ccc2e6f1180868ac6/manifest.json#L9)
This needs to be a valid URL

[.setTitle("Make it Professional")](https://github.com/manimohans/obsidian-local-llm-helper/blob/a80440c68f9a3a0c4a455c8ccc2e6f1180868ac6/main.ts#L53), [.setName("Server Address")](https://github.com/manimohans/obsidian-local-llm-helper/blob/a80440c68f9a3a0c4a455c8ccc2e6f1180868ac6/main.ts#L154), [.setName("Server Port")](https://github.com/manimohans/obsidian-local-llm-helper/blob/a80440c68f9a3a0c4a455c8ccc2e6f1180868ac6/main.ts#L168)
[Use sentence case in UI](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Use+sentence+case+in+UI)


[// If the plugin hooks up any global DOM events (on parts of the app that doesn't belong to this plugin)](https://github.com/manimohans/obsidian-local-llm-helper/blob/a80440c68f9a3a0c4a455c8ccc2e6f1180868ac6/main.ts#L87-L96)
Remove any code from the sample plugin that you don't need.

manimohans commented:
Made the following changes based on feedback:

- minAppVersion updated in manifest.json
- fundingURL updated
- Updated UI elements to sentence case.
- Removed sample plugin residual code.
- Also removed style.css file since it wasn't being used.
